docs: ✏️ added tutorial in moving contributors table (#554)
all-contributors/all-contributors: docs/bot/configuration.md

  "contributors": []
}
```

#### Moving the All Contributors table into a different file

As you know, the default file for the contributors table is `README.md`. But if your project has tons of contributors, it doesn't make sense to include the All Contributors table in the `README.md` file. Instead, you can use a different file to do so.

Here are the steps that may help you in achieving that. For the sake of this tutorial, the different file that we'll use is going to be named `CONTRIBUTORS.md`.

1. Install the bot. See the [Installation](installation) page for instructions.
2. Add your first contributor. See the [Bot Usage](usage) page for instructions.
3. Create the `CONTRIBUTORS.md` file.

> 4. Copy the code from the Contributors section in the `README.md` file, starting at the `ALL-CONTRIBUTORS-LIST:START` line, and ending at the `ALL-CONTRIBUTORS-LIST:END` line.

Reviewer: @hanisirfan Step 4 looks like this is already included in the current documentation. https://allcontributors.org/docs/en/bot/installation#3-create-a-readmemd

Reviewer: I guess we can just remove number step 4 and reference to this link mentioned above

    Example:

>     ```html

Reviewer 2: Nit: You don't need that indentation before the triple tildes (same goes for the lines below).

Author: So, all the code blocks doesn't need to be indented isn't it?

Author: ![image](https://user-images.githubusercontent.com/66242389/148619228-eaa3ad1f-f9a7-443c-9982-c1d046ef3e94.png)
after remove the indentations, it looks something like this

Reviewer 3: Yeah, the indentation is needed to make the code block (and the "Example" paragraph) a child of the list item. Indenting by 3 characters would be sufficient, though.

Author: @waldyrious I guess, I'll leave it with the current indentation.

Author: @waldyrious I've changed it from 4 spaces to 3. Please take a look at my latest commit 😁 Thanks

Reviewer 3: Hmm, you seem to have introduced tab-based indentation in the code blocks. Please re-check if that's what you intended.

Author: I actually doesn't know how to differentiate between tab-based and space-based since I only use tab for indentation. May you take a look at my current commit? And if you're using VS Code, can you share with me how to differentiate those 2 types of indentations? Thanks.

   <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
   <!-- prettier-ignore-start -->
   <!-- markdownlint-disable -->
   <table>
   <tr>
      <td align="center">...</td>
   </tr>
   </table>
   <!-- markdownlint-restore -->
   <!-- prettier-ignore-end -->
   <!-- ALL-CONTRIBUTORS-LIST:END -->
   ```
5. Modify the `files` key in the `.all-contributorsrc` JSON file to match your new file name.
   ```json
   {
   "files": [
      "CONTRIBUTORS.md"
   ],
   ...
   }
   ```

> 6. Now you should be good to go. New contributors will be added to this new file instead of the `README.md` file.

Reviewer: @hanisirfan I think Step 6 is not necessary. 

@Berkmann18 What do you think?

Reviewer 2: It could be removed, but it's a nice ending note as far as the steps go.

  "contributors": []
}
```

> #### Moving The All Contributors Table Into Different File

Reviewer 3: ```suggestion
#### Moving the All Contributors table into a different file
```

> As you know, the default file for the contributors table is `README.md`. But, if your project have tons of contributors, it doesn't make sense to includes the All Contributors Table in the `README.md` file. Instead, you can use a different file to do so.

Reviewer 3: ```suggestion
As you know, the default file for the contributors table is `README.md`. But if your project has tons of contributors, it doesn't make sense to include the All Contributors table in the `README.md` file. Instead, you can use a different file to do so.
```

> Here is the steps that may help you in achieving that. For the sake of this tutorial, the different file that we're going to be using is going to be named as `CONTRIBUTORS.md`.

Reviewer 3: ```suggestion
Here are the steps that may help you in achieving that. For the sake of this tutorial, the different file that we'll use is going to be named `CONTRIBUTORS.md`.
```

> 1. Install the bot. Read the [Installation]() page.

Reviewer 3: ```suggestion
1. Install the bot. See the [Installation]() page for instructions.
```

Reviewer 3: Also, the links appear to be missing the URL part inside the `()`.

Author: Oh for the links. I actually don't know how Docusaurus local links work. But, after some digging from the website source file, I think I know it now.

> 2. Add your first contributor. Read the [Bot Usage]() page.

Reviewer 3: ```suggestion
2. Add your first contributor. See the [Bot Usage]() page for instructions.
```

3. Create the `CONTRIBUTORS.md` file.

> 4. Copy the code from the Contributors section in the `README.md` file. Starting from the `ALL-CONTRIBUTORS-LIST:START` line and ending with `ALL-CONTRIBUTORS-LIST:END` line.

Reviewer 3: ```suggestion
4. Copy the code from the Contributors section in the `README.md` file, starting at the `ALL-CONTRIBUTORS-LIST:START` line, and ending at the `ALL-CONTRIBUTORS-LIST:END` line.
```

Example:
```
<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->
<table>
  <tr>
    <td align="center">...</td>
  </tr>
</table>

<!-- markdownlint-restore -->
<!-- prettier-ignore-end -->

<!-- ALL-CONTRIBUTORS-LIST:END -->

> ```

Reviewer 3: This entire block should be indented so it will appear as part of the item 4.

Author: Got it.

>     ```

Reviewer 2: Could you add the language used in the highlighted code?
I'm also unsure if that indentation is on the "Example:" and the backticks are necessary.

5. Modify the `files` key in the `.all-contributorsrc` JSON file to match your new file name.

Example:
```
{
  "files": [
    "CONTRIBUTORS.md"
  ],
  ...
}

> ```

Reviewer 3: Same as above — this entire block should be indented.

>     ```

Reviewer 2: ditto.
